packages/core/src/Game/GameObject/GameObject.js

```diff
@@ -874,7 +795,7 @@ GameObjectModule.TYPE = 'GameObject';
  * @param {number} ratio a number between 0 => 1
  * @returns {GameObject} g1 interpolated
  */
-GameObjectModule.interpolateInPlace = function (g1, g2, ratio) {
+GameObject.interpolateInPlace = function (g1, g2, ratio) {
```

Comment:
interpolate should interpolate browserscript/worldscript ? conf numeric values ?